main.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/valentindeaconu/terralist/api"
+	"github.com/valentindeaconu/terralist/database"
+	"github.com/valentindeaconu/terralist/service"
+)
+
+func main() {
+	if err := database.AppDatabase.Init(); err != nil {
+		panic(err.Error())
```

Comment:
Consider using [logrus](https://pkg.go.dev/github.com/sirupsen/logrus) for server type applications.